content/integrations/jira.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Datadog-Jira Integration
+integration_title: Jira
+kind: integration
+doclevel: basic
+---
```

Comment:
I just saw how what I said might be confusing. All the top level headers should be an h1, not just overview, but overview, installation, configuration, etc. as per the readme at the top of this repo